build_tester/tester.py

```diff
@@ -6,6 +6,7 @@
 import requests
 
 from build_tester.builders.docker_builder import DockerBuilder, DockerInfo
+from build_tester.builders.host_builder import HostInfo, HostBuilder
```

Comment:
Shouldn't it be in this order: `HostBuilder, HostInfo`.

Reply:
fixed